backend/user_service/controllers/userController.js

```diff
@@ -24,6 +23,30 @@ const addToUserCollection = async (req, res) => {
     }
 };
 
+const removeFromUserCollection = async (req, res) => {
+    const { uid } = req.body;
+    if (!uid) {
+        return res.status(400).json({ error: "User ID (uid) is required." });
```

Comment:
Change the field name from `error` to `message` to be in sync with how error messages are handled in `callFunction` in libs/util.ts.

Reply:
gdi i keep forgetting if its error or message, thanks for pointing it out